source/unified-test-format/unified-test-format.rst

```diff
@@ -2649,6 +2650,14 @@ to elsewhere in the specification as the internal MongoClient.
 Determine the server version and topology type using the internal MongoClient.
 This information will be used to evaluate any future `runOnRequirement`_ checks.
 
+In addition to the cluster URI, the test runner MUST also be configurable
+with two other connection strings (or equivalent configuration) that point to
+TCP load balancers: one fronting multiple servers and one fronting a single
+server. These will be used to initialize client entities when executing tests
+against a load balanced cluster. If the topology type is ``LoadBalanced``,
+the test runner MUST error if either of these URIs is not provided. For all
+other topology types, these URIs MUST be ignored.
```

Comment:
"For all other topology types, these URIs SHOULD NOT be provided and MUST be ignored if provided."

Alternatively, would it make sense to just say they "MUST NOT be provided", or are you thinking that some CI configurations need to set these environment variables as empty strings (or equivalent)?

Reply:
The latter. At least for Go, we don't usually do empty string checks to determine if certain environment variables should be passed into the test runner. Instead, we'll just access the ones we know _can_ exist and they'll be empty strings that get ignored by the runner if it doesn't need them.

I updated the text to "For all other topology types, these URIs SHOULD NOT be provided and MUST be ignored if provided".